### DISCUSSION TITLE
Tasmota flashed on Zigbee Bridge Pro sometimes becomes unresponsive when internet connection drops

### DISCUSSION THREAD

**Discussion by msehome** [arendst/Tasmota#22297] [Q&A] [answered]
Sonoff Zigbee Bridge Pro flashed with Tasmota occasionally becomes unresponsive when internet connection drops!
It sometimes recovers itself but often needs a power cycle to reboot. Rebooting via Web interface does not fix the issue. It may also stop responding to pings!
The usual recovery process is to disconnect and reconnect the power cord of the bridge and restart the Zigbee2mqtt addon as well.
I was wondering how the Sonoff Zigbee Bridge Pro is dependent on internet connection and why it should become unstable after internet is down?

**Comment by Jason2866** [comment]
Tasmota needs to reach a NTP server. If it is a Internet NTP server there is the problem.
You can setup a NTP server in your LAN.

**Reply by msehome** [reply]
Good point, So you believe that may fix it. I will give it a shot and report back but where to set the NTP server or will it detect it automatically or is there a way to disable it?
And another point here. What if we need to use the bridge on a network with no internet what so ever?

**Reply by s-hadinger** [reply, marked as answer]
Internet access is not required for anything in Tasmota. As mentioned by Jason, the only hard requirements is NTP. You can host your local NTP server, it does not even require to be atomic-clock accurate.

**Reply by msehome** [reply]
Ok great, but I didn't get whether Tasmota detects the local NTP server automatically?

**Reply by s-hadinger** [reply]
Not automatically, you need to specify with the command `NtpServer`

**Reply by msehome** [reply]
Ok thank you! You are doing an amazing job! Appreciate your effort and perseverance!
